tetrateci/tsb-util/templates/k8s-objects/bridged/servicerouteeditor.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ servicerouteSAName }}
+  namespace: tsb
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ podName }}
+  namespace: tsb
+spec:
+  containers:
+  - image: amd64/ubuntu:focal
+    name: tctl-container
+    command: [ "/bin/bash", "-c", "--" ]
+    args:
+    - |
+      apt update && apt install curl -y
+      curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+      install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+      IP=$(kubectl get svc envoy -n tsb -o "jsonpath={.status.loadBalancer.ingress[0].ip}")
```

Comment:
This could be hostname if the k8s provider is AWS. Can we take the k8s provider name from config yaml? If provider name is NOT AWS, then use IP otherwise 'hostname'? Default is IP

Reply:
hmm that is something I didn't think of